nebula_messages/continental_msgs/msg/ContinentalSrr520DetectionList.msg

```diff
@@ -0,0 +1,10 @@
+std_msgs/Header header
+uint32 internal_time_stamp_usec
+uint8 global_time_stamp_sync_status
+uint8 signal_status
+uint8 sequence_counter
+uint32 cycle_counter
+float32 vambig # cSpell:ignore vambig
```

Comment:
`vambig` was added to .cspell so we don't need to ignore here anymore.
```suggestion
float32 vambig
```

Reply:
Addressed in [54b75c4](https://github.com/tier4/nebula/pull/124/commits/54b75c465c78af05d71bceeb3b20a46631db0da3)